README.md

```diff
@@ -34,17 +34,54 @@ See below for more details.
 ## Setup
 
 * [Computer setup](https://github.com/panoptes/POCS/wiki/Panoptes-Computer-Setup)
+* While logged in as user panoptes:
+   * Create /var/panoptes, owned by user panoptes:
+     ```bash
+     sudo mkdir -p /var/panoptes
+     sudo chown panoptes /var/panoptes
+     chmod 755 /var/panoptes
+     mkdir /var/panoptes/logs
+     ```
+   * Define these environment variables, both in your current shell and in
+     `$HOME/.bash_login` (to only apply to user panoptes) or in `/etc/profile`
```

Comment:
I read up a little more about the execution order of these (which I should have done 10 years ago) and it seems like this should be `.bash_profile` rather than `.bash_login`. And I'm not sure we need to mention the option for all users although I guess doesn't hurt.